app/controllers/GenericExplorerController.scala

```diff
@@ -96,6 +96,25 @@ class GenericExplorerController @Inject() (
     future.map( i => Ok( Json.toJson( i ) ) )
   }
 
+  //Get all edges belonging to a node
+  def retrieveNodeEdges( id: Long ) = ProfileFilterAction( jwtVerifier.get ).async { implicit request =>
```

Comment:
Can we include filters for in/out? see https://github.com/SwissDataScienceCenter/renga-graph/blob/master/navigation/service/app/controllers/EdgeController.scala#L104-L135 for reference.

Reply:
Indeed could be nice to have. However, currently the edges direction has no real semantic (if you change the edge label you can easily invert the direction). (stores / stored_in).
But to be future-proof, we can add it.